IyiKuyoro/Pals
# #161530821: Setup express server

#### What does this PR do?
Setup the express server

#### Tasks to be completed
> - Install the express package
> - Setup express server
> - Transpile code down to es5 with babel for both development and production

#### How can this be manually tested?
> - Clone this repository
> - Checkout to `ch-setup-express-server-#161530821`
> - Start the `dev` server by running `npm run dev` or the production server by running `npm start`
> - Navigate to `localhost:3000` or the port you have declared in your `.env` file.
> - a JSON response with a message should show.

#### Relevant PT stories
#161530821
